challenge-04/README.md

```diff
@@ -7,16 +7,42 @@ um único parâmetro como argumento. Essa função deve retornar `true` se o
 equivalente booleano para o valor passado no argumento for `true`, ou `false`
 para o contrário.
 */
-?
+var isTruthy = function(arg) {
+	if(arg) {
+		 return true;
+	} else {
+		return false;
+	}
+};
+/* 
+Eu tentei fazer utilizando condicional ternario dessa maneira:
+var isTruthy = function(arg) {
+   arg ? true : false;
+};
+mas ele retorna apenas undefined.
```

Comment:
@silvioCoelho é porque não há retorno na função. Você simplesmente tenta imprimir `true` se `arg` for definido, e `false` caso contrário. Para fazer qualquer função retornar valores diferente de `undefined`, você precisa usar o `return` ;)